explainer-server/public/stylesheets/main.css

```diff
@@ -212,5 +212,11 @@ body {
   border-left: 1px solid #bdbdbd;
 }
 
+#top-toolbar__word-count-wrapper {
+  border-left: 1px solid #bdbdbd;
+  padding:0px 15px 0px 15px;
+}
 
-
+#top-toolbar__word-count-wrapper__count {

```

Comment:
Try not to use id selectors in CSS> The specificity is too high and causes problems later on.